src/EurekaClient.js

```diff
@@ -303,19 +303,21 @@ export default class Eureka {
    */
   transformRegistry(registry) {
     if (!registry) {
-      throw new Error('Unable to transform empty registry');
-    }
-    this.cache = { app: {}, vip: {} };
```

Comment:
Considering this function is entirely synchronous, I'm guessing it was an unrelated issue.